src/index.ts

```diff
@@ -45,8 +62,9 @@ export default class IncludeFragmentElement extends HTMLElement {
     this.setAttribute('accept', val)
   }
 
+  // TODO: Should this return a TrustedHTML if available, or always a string?
```

Comment:
I could use advice on this. On the one hand, it would be less surprising (and backwards-compatible) to always return a string.

However, that string can't be assigned to another `.innerHTML` without going through a CSP trusted types policy again.

(Copied from an older comment.)

Reply:
We can always mark this as a breaking change and return `TrustedHTML`

Reply:
We discussed this in team pairing and replaced it with a comment, because we don't need it right now. Does that sound alright to you?